test_cases/example_2.json

```diff
@@ -1,1164 +1,925 @@
+                    "BallotName": {
+                        "Text": [
+                            {
+                                "Content": "Anthony Alpha",
+                                "Language": "en",
+                                "@type": "ElectionResults.LanguageString"
+                            }
+                        ],
+                        "@type": "ElectionResults.InternationalizedText"
+                    },
+                    "IsTopTicket": true,
+                    "PartyId": "lepton_party",
+                    "@type": "ElectionResults.Candidate"
+                },
+                {
+                    "@id": "candidate_gamma",
+                    "BallotName": {
+                        "Text": [
+                            {
+                                "Content": "Gloria Gamma",
+                                "Language": "en",
+                                "@type": "ElectionResults.LanguageString"
+                            }
+                        ],
+                        "@type": "ElectionResults.InternationalizedText"
+                    },
+                    "IsTopTicket": true,
+                    "PartyId": "hadron_party",
+                    "@type": "ElectionResults.Candidate"
+                },
+                {
+                    "@id": "candidate_beta",
+                    "BallotName": {
+                        "Text": [
+                            {
+                                "Content": "Betty Beta",
+                                "Language": "en",
+                                "@type": "ElectionResults.LanguageString"
+                            }
+                        ],
+                        "@type": "ElectionResults.InternationalizedText"
+                    },
+                    "PartyId": "lepton_party",
+                    "@type": "ElectionResults.Candidate"
+                },
+                {
+                    "@id": "candidate_delta",
+                    "BallotName": {
+                        "Text": [
+                            {
+                                "Content": "David Delta",
+                                "Language": "en",
+                                "@type": "ElectionResults.LanguageString"
+                            }
+                        ],
+                        "@type": "ElectionResults.InternationalizedText"
+                    },
+                    "PartyId": "hadron_party",
+                    "@type": "ElectionResults.Candidate"
+                }
+            ],
+            "BallotStyle": [
+                {
+                    "@type": "ElectionResults.BallotStyle",
+                    "GpUnitIds": ["precinct_1_downtown"],
+                    "OrderedContent": [
+                        {
+                            "@type": "ElectionResults.OrderedContest",
+                            "ContestId": "contest-orbit-city-mayor",
```

Comment:
My parser encountered the following error here:

```
Unable to find a contest for contestId contest-orbit-city-mayor
```

It looks like the error is correct - there is no contest ID of `contest-orbit-city-mayor` elsewhere in this EDF, so the code cannot do a lookup to fetch more information.

I made an educated guess and changed this highlighted line to `contest_1` (so that it would match the contest ID) and it worked! Unfortunately, it then failed due to the next comment in this review.

Reply:
global change of contest_1 to contest-orbit-city-mayor